src/main/content/_data/en/en.yml

```diff
@@ -123,4 +123,5 @@ instance:
   no-apps: Deploy your application to begin managing them.
   app-hub: Application Hub
   oauth:
-    configure: Configure OAuth
\ No newline at end of file
+    configure: Configure OAuth
+  cli-version: "CLI Version: "
```

Comment:
does this need the double quotes?

Reply:
Yeah the colon causes jekyll to throw an error so anytime we have a special character in the en.yml file we have to put quotes aroung it. We can remove it from the en.yml file and hard code the colon in the html which would allow us to take out the double quotes.